examples/web-wagmi/src/profiles/test.tsx

```diff
@@ -0,0 +1,19 @@
+import { ProfileFieldsFragment, useFollow } from '@lens-protocol/react';
+
+type ProfileFollowProps = {
+  profile: ProfileFieldsFragment;
+};
+
+export function FollowProfile({ profile }: ProfileFollowProps) {
```

Comment:
was this file intended to be added to the repo? (test.tsx)

Reply:
thanks, removed